scripts/overnight-conformance.sh

```
@@ -18,6 +18,10 @@ LOG="$RESULTS_DIR/run-$TIMESTAMP.log"
 SERVER_PID=""
 
 mkdir -p "$RESULTS_DIR"
+if [[ -z "${PRELOOP_SYSTEM_TOKEN:-}" ]]; then
+    export PRELOOP_SYSTEM_TOKEN="$(python3 -c 'import secrets; print(secrets.token_hex(32))')"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)